config/samples/xgboost-dist/README.md

```diff
@@ -248,3 +249,242 @@ Events:
   Normal  ExitedWithCode           38s                xgboostjob-operator  Pod: default.xgboost-dist-iris-test-predict-master-0 exited with code 0
   Normal  XGBoostJobSucceeded      38s                xgboostjob-operator  XGBoostJob xgboost-dist-iris-test-predict is successfully completed.
 ```
+
+**Start the distributed XGBoost train to store the model locally**
```

Comment:
Probably highlight the difference in the YAMLs between OSS and local to be PVC and usage of parameter model_storage_type as local. Default is OSS.

Reply:
done